lms/djangoapps/certificates/views/webview.py

```diff
@@ -416,10 +417,12 @@ def render_html_view(request, user_id, course_id):
         )
     )
     context['share_url'] = share_url
-    twitter_url = 'https://twitter.com/intent/tweet?text={twitter_share_text}&url={share_url}'.format(
-        twitter_share_text=context['twitter_share_text'],
-        share_url=urllib.quote_plus(share_url)
-    )
+    twitter_url = ''
+    if context.get('twitter_share_enabled', False):
+        twitter_url = 'https://twitter.com/intent/tweet?text={twitter_share_text}&url={share_url}'.format(
+            twitter_share_text=smart_str(context['twitter_share_text']),
```

Comment:
@ziafazal what do you think if we follow this same approach for facebook share text as well ?